care/emr/api/viewsets/report/template.py

```
@@ -0,0 +1,152 @@
+    @action(detail=False, methods=["GET"], url_path="schema")
+    def get_schema(self, request, *args, **kwargs):
+        return Response({})
+
+    @extend_schema(
+        description="Preview a template with sample data",
+        request=PreviewTemplateRequest,
+        responses={200: "Success"},
+        tags=["template"],
+    )
+    @action(detail=False, methods=["POST"])
+    def preview(self, request, *args, **kwargs):
```

Ruff (0.14.6)

Unused method argument: `args`

(ARG002)